packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/description/DiagramDescription.java

```diff
@@ -104,6 +107,10 @@ public List<EdgeDescription> getEdgeDescriptions() {
         return this.edgeDescriptions;
     }
 
+    public List<ITool> getTools() {
+        return this.tools;
+    }
```

Comment:
The `DiagramDescription` is not to be used as a delivery boy anymore than it is already. If you need to ask for a reconnection tool in the `ViewReconnectionToolsExecutor` then you should figure it out there. Don't add this data structure here only as a shortcut for the View DSL support.